ts/src/index.ts

```diff
@@ -3,17 +3,26 @@ import '@babel/polyfill';
 import * as _ from 'lodash';
 import 'reflect-metadata';
 
+import { configs as testConfigs } from '../test/test_configs';
```

Comment:
`testConfigs` are not used in this file.

Reply:
Hm I thought our convention was to put all exports in the index.ts. Should I just export as `testConfigs` and `productionConfigs` (not nested)?